### ISSUE TITLE
ctp1 IEA query

### ISSUE THREAD

**Issue by pombase-admin** [pombase/curation#587]
I notice in the new gaf that ctp1 has:

PomBase	SPCC338.08	ctp1		GO:0000077	GO_REF:0000002	IEA	InterPro:IPR013882	P	CtIP-related endonuclease	mug38|slr9|nip1	protein	taxon:4896	20150328	InterPro

Does this not pass the 'rules' for annotating to a checkpoint

I see it has "abnormal DNA damage checkpoint" FYPO annotations.... does this mean that the checkpoint is activated or not activated ?...I think ctp1 possibly when mutated causes a problem which is detected by the checkpoint...does that make sense?

Original comment by: ValWood

**Comment by ValWood** [comment]
more from a recent e-mail. Antonia asked the same question
-

yep I have been surveying that one as its in
http://curation.pombase.org/dumps/latest_build/logs/log.2015-10-26-22-24-44.excluded_go_terms

Based on the paper Interpro use to make the mapping its kind of difficult to argue that it shouldn't
http://europepmc.org/abstract/MED/17965729

I'm not sure whether its really involved in the checkpoint or its causing the checkpoint to be activated when mutated.... needs further investigation to give feedback on the mapping. I think it is most likely indirect?

The question is, does the mutated ctp1 *result in checkpoint activation* (i.e causing a problem which is sensed by the checkpoint machinery, in which case it should *not* have a checkpoint GO annotation or is it *required for the checkpoint signalling to occur*.....

val

**Comment by mah11** [comment]
PMID:17936710 is on my list for the MRE11/MRN complex, but it'll be a while before I get round to curating it fully (it's a dauntingly long list :/ ). I did have a quick look, which suggests that ctp1 deletion causes a problem that the checkpoint detects, at least in pombe:

"Microscopic examination revealed that ctp1Δ cells arrest division in response to IR (Fig. 1B), hence their DNA damage checkpoint is intact. This conclusion was confirmed by immunoblot analysis of the DNA damage checkpoint kinase Chk1 (Fig. 1C), which is hyper-phosphorylated when activated."

But because this stuff is never simple and convenient, it's possible that the situation in human is more complicated. I skimmed PMID:17965729 even more superficially, but it includes this:

"Notably, CtIP depletion markedly impaired Chk1 phosphorylation at late time-points but not at early time-points after continuous camptothecin exposure (Supplementary Fig. 1e). This suggested that CtIP is not required for initial Chk1 phosphorylation triggered by replicational stress induced by trapped DNA topoisomerase I molecules but is needed for signaling of camptothecin-induced replication-dependent DSBs, ..."

So right now it does look like we should have only phenotype annotations, but I don't have enough to rule out the possibility that a checkpoint GO annotation wouldn't be as wrong for human CtIP as it would be for pombe ctp1.

Is that what you wanted for this ticket?
